docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -242,6 +233,76 @@ This header has several directives for fine-tuning the referrer information.
+
+### Enable HTTP Strict Transport Security (HSTS)
+
+HSTS forces clients to always communicate with your site over HTTPS.
+[Most browsers support this](https://caniuse.com/stricttransportsecurity), and there is no downside for browsers that don't.
+Read the requirements and instructions at [hstspreload.org](https://hstspreload.org/) before you enable HSTS.
+Make sure to also include subdomains using the `includeSubDomains` setting.
+
+When using [[= product_name_cloud =]], you can [configure HSTS in .platform/routes.yaml](https://docs.platform.sh/define-routes/https.html#enable-http-strict-transport-security-hsts).
+
+## Domain
+
+### Enable Domain Name System Security Extensions (DNSSEC)
+
+DNSSEC is a DNS feature that authenticates responses to DNS requests.
+It protects against DNS poisoning attacks, which is when an attacker manipulates the reponses to DNS requests with the goal of directing users to an IP address the attacker controls.
+Enabling DNSSEC involves creating the DNSSEC records in your domain, activating DNSSEC with your domain registrar, and enabling DNSSEC signature validation on all DNS servers.
+[Read more on DNSSEC at ICANN](https://www.icann.org/resources/pages/dnssec-what-is-it-why-important-2019-03-05-en).
+
+### Enable domain update/delete protection
+
+Domain update/delete protection is a DNS setting that makes it harder for an attacker to take over a domain from the real owner, or hinder availability for users.
+You can enable this protection at your domain registrar's site.
+Log in to their site to enable these protection settings and save the new configuration.
+
+### Enable Certificate Authority Authorization (CAA)
+
+CAA allows domain owners to specify which Certificate Authorities (CAs) are permitted to issue SSL/TLS certificates for their domain.
+This prevents attackers from having certificates issued for domains they don't own, hindering some types of attack.
+CAA is configured in your DNS zone file.
+
+## Database
+
+### Use UTF8MB4 with MySQL/MariaDB
+
+If you're using MySQL/MariaDB, use the UTF8MB4 database character set and related collation.
+The older UTF8 can lead to truncation with 4-byte characters, like some emoji, which may have unpredictable side effects.
+
+See [Change from UTF8 to UTF8MB4](update_db_to_2.5.md#change-from-utf8-to-utf8mb4).
+
+### Secure access
+
+Secure the database access with strong passwords, keys, firewall, encryption in transit, encryption at rest etc. as needed.
+When using [[= product_name_cloud =]], the provider handles this.
+
+### Limit database rights
+
+Optionally, ensure that the database user used by the web app only has permissions to do the operations needed by [[= product_name =]].
+The Data Definition Language (DDL) commands (create, alter, drop, truncate, comment) are only needed for installing and upgrading [[= product_name =]], and not for running it.
+Not granting these rights to web app users reduces the damage that can result from a security breach.
```

Comment:
So, what at the rights to give to web app users to have the needed ones and only the needed ones? (see #2356)

Reply:
@adriendupuis This isn't documented AFAIK. Ref. the doc that only says "Before executing the command make sure that the database user has sufficient permissions."
https://doc.ibexa.co/en/latest/getting_started/install_ibexa_dxp/#create-a-database

Incidentally, I found that the next paragraph doesn't belong there. PR to move it: https://github.com/ibexa/documentation-developer/pull/2569

Anyway, I removed the section, and instead added a new Monitor section at the end, about monitoring the various part of the system. See https://github.com/ibexa/documentation-developer/pull/2565/commits/26d5a78bbefe56e76027093f57f525353420657f